sys/psa_crypto/include/psa_ciphers.h

```diff
@@ -127,7 +132,57 @@ psa_status_t psa_cipher_cbc_aes_256_encrypt(const psa_key_attributes_t *attribut
                                             uint8_t *output,
                                             size_t output_size,
                                             size_t *output_length);
+#endif /* CONFIG_CIPHER_CBC_AES_256 */
 
+#if IS_USED(MODULE_PSA_CIPHER_CHACHA_20) || defined(DOXYGEN)
+
+/**
+ * @brief   Low level wrapper function to call a driver for ChaCha20 encryption/decryption.
+ *          See @ref psa_cipher_encrypt().
+ * @param key_buffer        Buffer containing the Key.
+ * @param key_buffer_size   Size of the key in bytes.
+ *                          Must be of size CHACHA20POLY1305_KEY_BYTES.
```

Comment:
```suggestion
 *                          Must be equal to CHACHA20POLY1305_KEY_BYTES.
```

Also, maybe don't refer to the software-backend defined constants here, but just state the concrete numbers.